benchmarking/utils/dict_get.py

```diff
@@ -0,0 +1,15 @@
+
+
+def dict_get(params: dict, key: str, default):
```

Comment:
Very last little ask. Could you import this and get_cost_model in benchmarking/utils/__init__.py? Otherwise, importing this is very annoying.